activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb

```diff
@@ -296,7 +296,7 @@ def create_table(table_name, id: :primary_key, primary_key: nil, force: nil, **o
         td = build_create_table_definition(table_name, id: id, primary_key: primary_key, force: force, **options, &block)
 
         if force
```

Comment:
Should we move the check for `if_not_exists` up?
```suggestion
        if force && !td.if_not_exists
```

Previously, if `:force` and `:if_not_exists` were both set, we would've dropped the table and cleared the schema cache. If only `:if_not_exists` was set, we would've fallen into the `else` branch and cleared the schema cache. Moving the check up ensures that we still fall into the `else` branch and clear the schema cache.